vultr/resource_block_storage.go

```diff
@@ -0,0 +1,169 @@
+package vultr
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceBlockStorage() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBlockStorageCreate,
+		Read:   resourceBlockStorageRead,
+		Update: resourceBlockStorageUpdate,
+		Delete: resourceBlockStorageDelete,
+
+		Schema: map[string]*schema.Schema{
+			"subid": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"label": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"dcid": {
+				Type:     schema.TypeInt,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"size_gb": {
```

Comment:
What do you think about renaming this to simply “size”. I think it is a little more intuitive for a required field name